etcd3/utils.py

```diff
@@ -382,3 +382,12 @@ def find_executable(executable, path=None):  # pragma: no cover
             f = os.path.join(p, execname)
             if os.path.isfile(f):
                 return f
+
+
+class EtcdEndpoint():
```

Comment:
this class contains only host and port but made creating a client less friendly

any further design on this?

Reply:
aiohttp.ClientError    requests.RequestException   urllib3.exceptions.HTTPError